webapp/src/sagas.js

```diff
@@ -67,28 +67,41 @@ function* rootSaga() {
 
   yield takeLatest(types.subscribeEmail.success, handleAddressFetchReload)
 
-  yield put({ type: types.connectWeb3.request })
+  yield takeLatest(types.fetchStats.request, handleStatsFetchRequest)
+  yield takeLatest(
+    types.fetchAddressStats.request,
+    handleAddressStatsFetchRequest
+  )
 }
 
 // -------------------------------------------------------------------------
 // Web3
 
-async function connectLedger(action = {}) {
+async function connectLedger(action = {}, retries = 0) {
   try {
     if (window.navigator.userAgent.toLowerCase().indexOf('firefox') > -1) {
       return false
     }
+
     const ledger = window.ledger
     const comm = await ledger.comm_u2f.create_async(5)
```

Comment:
Careful! This 5 means `5 seconds`. It will slow down 15 seconds given that the saga expects this to finish before entering the auction

Reply:
Auch, didn't notice that wile testing. That's not acceptable.
You think that keeping this approach and lowering the delay would work?

(thanks for looking at this)

Reply:
I think we don't have to retry with the ledger!